packages/maskbook/src/plugins/ITO/types.ts

```diff
@@ -23,8 +23,6 @@ export interface JSON_PayloadInMask {
     token: EtherTokenDetailed | ERC20TokenDetailed
     exchange_amounts: string[]
     exchange_tokens: (EtherTokenDetailed | ERC20TokenDetailed)[]
-    is_mask?: boolean
```

Comment:
since they're encoded into the payload, I think you should comment them out and we won't accidentally use them in the future in an incompatible way.